api/v1alpha1/sandbox_types.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2025.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+// Important: Run "make" to regenerate code after modifying this file
+
+// SandboxSpec defines the desired state of Sandbox
+type SandboxSpec struct {
+	// The following markers will use OpenAPI v3 schema to validate the value
+	// More info: https://book.kubebuilder.io/reference/markers/crd-validation.html
+
+	// State: RUNNING(default)|SUSPENDED
+	// +kubebuilder:default="RUNNING"
+	// +kubebuilder:validation:Enum=RUNNING;SUSPENDED
+	// +kubebuilder:validation:Required
+	State string `json:"state" protobuf:"bytes,1,opt,name=state"`
+
+	// template is the object that describes the pod spec that will be used to create
+	// an agent sandbox. Each pod will be named with the format sandbox.metadata.name
```

Comment:
Suggest removing "Each pod will be named with the format sandbox.metadata.name" from the doc. This is an implementation detail and is subject to change (e.g. what if we don't create pods directly, or needs a different way to handle pod name conflicts?)

Reply:
makes sense will remove.